packages/salesforcedx-vscode-core/package.json

```diff
@@ -111,6 +111,11 @@
         {
           "command": "sfdx.force.task.stop",
           "when": "view == sfdx.force.tasks.view"
+        },
```

Comment:
A heads up- the api to display the icon inline only displays the icon when the user is hovering over it.